src/API/Surface.hs

```diff
@@ -83,17 +78,6 @@ loadTheory config isTPTP = case configInput config of
             Right (Input thy dps) -> do
               let config' = config {configSkolemDepth = dps}
               return $ Right (config', thy)
-  --where
```

Comment:
I don't remember why this was committed. Eventually, we need to be able to load TPTP files.